### PR TITLE
Add button on user profile to display token

### PR DESCRIPTION
**Pull request by ryzokuken**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

@jywarren take a look

### CODE REVIEW

File: app/views/users/profile.html.erb [hunk 1]
```diff
@@ -213,6 +215,9 @@
 </div>
 
 <script type="text/javascript">
+  $('#btn-token').click(function() {
+    alert("Your access token is: <%= @user.user.token %>");
```
Comment (jywarren) [inline app/views/users/profile.html.erb]: If you use a prompt it can be selectable?

File: config/routes.rb [hunk 2]
```diff
@@ -31,8 +31,8 @@
   # switch off subdomain matching when in development
   if Rails.env.test?
   # or to skip www:
-    match "", to: 'wiki#subdomain', constraints: lambda { |r| r.subdomain.present? && r.subdomain != 'www' && r.subdomain != "i" && r.subdomain != "test" && r.subdomain != "new" && r.subdomain != "alpha"}
-    match "*all", to: 'wiki#subdomain', constraints: lambda { |r| r.subdomain.present? && r.subdomain != 'www' && r.subdomain != "i" && r.subdomain != "test" && r.subdomain != "new" && r.subdomain != "alpha"}
+  match "", to: 'wiki#subdomain', constraints: lambda { |r| r.subdomain.present? && r.subdomain != 'www' && r.subdomain != "i" && r.subdomain != "test" && r.subdomain != "new" && r.subdomain != "alpha"}
```
Comment (jywarren) [inline config/routes.rb]: Are these changes necessary? This indentation in particular seems to belong the way it was. 

But otherwise this looks great! Can u share a screenshot of the code being shown?

Comment (ryzokuken) [inline config/routes.rb]: sure, I'll reset these particular changes if you want to.

Comment (ryzokuken) [inline config/routes.rb]: oh, wait. I didn't see the `if` block. Will do.

File: app/views/users/profile.html.erb (hunk 1 above)
Comment (ryzokuken) [inline app/views/users/profile.html.erb]: It can still be selected. However, we could later add helpful "copy to clipboard" shortcuts.

**Comment by ryzokuken** [comment]
screenshot_2017-07-14_03-39-32

**Comment by jywarren** [comment]
Ok great. Can we put this at the bottom and make it a white button? Then
we're good to merge!

**Comment by ryzokuken** [comment]
I did it this way to save one comparison :P
Will put it down there.

**Comment by ryzokuken** [comment]
@jywarren done. Here's the screenshot
screenshot_2017-07-14_04-17-36

**Comment by jywarren** [comment]
:-) great!

**Comment by ryzokuken** [comment]
Thanks. Now that this has been taken care of, tomorrow let's make an
account for the bot on publiclab.org and start working on some interesting
behaviours.

**Comment by jywarren** [comment]
Super! Please test with the version on staging for now. And please test on
a comment thread that won't send notifications to too many people.

Have fun! :-)
